plone/scale/tests/test_scale.py

```diff
@@ -219,6 +217,58 @@ def testResultBuffer(self):
         self.assertEqual(result, img2)      # the return value _is_ the buffer
         self.assertEqual(result.getvalue(), img1)   # but with the same value
 
+    def testAlternativeSpellings(self):
+        """Test alternative and deprecated mode spellings and the old
+        ``direction`` arguments instead of ``mode``.
+        """
+        
+        # scale-crop-to-fit
+        img = PIL.Image.new('RGB', (20, 20), (0, 0, 0))
+        img_scaled = scalePILImage(img, 10, 5, direction='scale-crop-to-fit')
+        self.assertEqual(img_scaled.size, (10, 5))
+        # down
+        img = PIL.Image.new('RGB', (20, 20), (0, 0, 0))
```

Comment:
By the way: I noticed that the ``scalePILImage`` method scales the image in place. If possible, this should only return a new image while keeping the old one as-is.
Not part of this PR though.

Reply:
See: https://github.com/plone/plone.scale/issues/38